PICMI_Python/particles.py

```diff
@@ -53,10 +53,7 @@ def __init__(self, particle_type=None, name=None, charge_state=None, charge=None
         self.handle_init(kw)
 
     def activate_field_ionization(self, model, product_species):
```

Comment:
Thanks for submitting this PR. My main suggestion is that this routine should be generic, something like
```
def add_interaction(self, interaction):
    self.interactions.append(interaction)
```
The user would then create the instance for the type of interaction and pass it in
```
field_ionization = picmi.FieldIonization('ADK', [ions, electrons])
neutrals.add_interaction(field_ionization)
```

Another suggestion is that interactions be specified in the simulation object. Then it would be more like
```
neutral_field_ionization = picmi.FieldIonization('ADK', neutrals, [ions, electrons])
sim.add_interaction(neutral_field_ionization)
```
This would make it more generalizable to other interactions where multiple species interact, such as binary collisions and fusion reactions. The classes for the interactions would be defined in a new file, interactions.py.